lib/router/selectedProduct.js

```diff
@@ -1,12 +1,12 @@
-FlowRouter.route('/ProductsExample', {
+FlowRouter.route('/SelectedProduct', {
     // Render the 'page'
     action: function(params, queryParams) {
       BlazeLayout.render('layout', {
         header:'header',
         body: 'main',
-        main: "productsExample"
+        main: "selectedProduct"
       });
     },
 
-    name: "productsExample" // optional
-});
+    name: "selectedProduct" // optional
+});
```

Comment:
Odd, mine usually does too. Wonder why it didn't this time. Funnily enough, i just opened the file on my computer, immediately hit save and it added the empty line at the bottom.